openapi.yaml

```diff
@@ -3274,22 +3099,50 @@ components:
           $ref: '#/components/schemas/generic-identifier'
         name:
           $ref: '#/components/schemas/generic-identifier'
+
+    temporary-rating:
+      type: object
+      additionalProperties: false
+      description: |
+        Data structure for a temporary rating against a segment.
+        Includes a unique ID, start and (optional) end time, and a reason.  
+
+        May include a set of values, which apply if this temporary rating
+        is either an AAR Exception or temporary seasonal rating.  
+
+      properties:
+        id:
```

Comment:
re: linting convo, some of these will be required and others not. This is obviously got to be there after creation but not on submission when it gets POST'd. I don't have the time atm to double check that we've addressed that, and perhaps it can wait. I'll let you be the judge.